getting_started/escoria_nodes.rst

```diff
@@ -0,0 +1,135 @@
+====================
+
+To make games in Escoria it's handy to understand the relationship between
+Godot's Scenes and nodes. These are documented here
+https://docs.godotengine.org/en/stable/getting_started/step_by_step/nodes_and_scenes.html#nodes
+
+Every location the game character can go to in the game will be its own Godot
+scene, and each of these scenes is referred to as a Escoria `room`.
+An Escoria `room` might be a bedroom or bathroom, but might also be the
+roadway on a bridge, or a forrest clearing. All the objects in the room
+are Godot nodes.
+
+There are several different types of Godot nodes. Their names and a brief
+overview of their uses is detailed below. For further information on each
+configurable parameter for each node, what it is and what it is used for,
+see the associated API page.
+
+.. hint::
+
+    Where an API page says `Extends: <sometype>`, this means that the
+    particular Escoria node has inherited all the features of the node type
+    "sometype" and builds on them. Any additional features it has will be
+    described in the "Property Description" of the node on the API page.)
+
+    For example, ESCRoom `Extends: Node2D`. This means it has all of Node2D's
+    features, like a "position" and a "scale". The API page also tells you
+    in its "Property Description" that it has (among other things) an
+    associated "player_scene" and "camera_limits" which can be configured.
+
+Most node types referenced in the API are intended to be used internally by
+Escoria. The ones of most use to game makers are :
+
+ESCRoom
+~~~~~~~
+
+An ESCRoom is the root node of your room scene. This is responsible for telling
+Escoria which character you'll control in the room, the script to use to
+configure the room before use, and the area of the room the camera is allowed
+to see.
+
+.. _`ESCRoom`: https://docs.escoria-framework.org/en/devel/api/ESCRoom.html
+
+ESCItem
+~~~~~~~
+
+An ESCitem is an object that the player can interact with in some way, a
+window or a phone for example. You associate an Escoria script with the item
+to tell Escoria what to do when the player interacts with the object (what
+happens when they "look at" or "use" the object for example.
+
+An ESCItem is accessible in Escoria scripts by the name provided in the
+`Global ID` field. This name should be unique to your whole game.
+
+You will typically assign a few child nodes to each ESCItem.
+
+* Sprite / Animated Sprite : This is an image of the object that will be
+  displayed on the screen for the player.
+* CollisionPolygon2D : This is a shape that tells Escoria when the mouse is
+  over the object. Set this up to cover the Sprite.
+* ESCLocation : If you make an ESCLocation the child of an ESCItem, when you
+  interact with the item, the character will automatically walk to this
+  position to perform the interaction. eg. place an ESCLocation underneath a
+  "window" ESCItem and the player will walk to the window when you "look" at
+  it.
```

Comment:
We need to stop rather than pause before the "e.g.", the sentence is too long otherwise.
What about this?
```suggestion
* `ESCLocation` : If you make an `ESCLocation` the child of an `ESCItem`, when you
  interact with the item the character will automatically walk to this
  position to perform the interaction. As an example, place an `ESCLocation` 
  underneath a "window" `ESCItem` and the player will walk to the window 
  when you "look" at it.
```

Reply:
Works for me!